fuse_examples/tutorials/hello_world/hello_world_utils.py

```diff
@@ -0,0 +1,36 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+def perform_softmax(output):
+    if isinstance(output, torch.Tensor):  # validation
+        logits = output
+    else:  # train
+        logits = output.logits
+    cls_preds = F.softmax(logits, dim=1)
+    return logits, cls_preds
+
+class LeNet(nn.Module):
+    def __init__(self):
+        super(LeNet, self).__init__()
+        self.conv1 = nn.Conv2d(1, 32, 3, 1)
+        self.conv2 = nn.Conv2d(32, 64, 3, 1)
+        self.dropout1 = nn.Dropout(0.25)
+        self.dropout2 = nn.Dropout(0.5)
+        self.fc1 = nn.Linear(9216, 128)
+        self.fc2 = nn.Linear(128, 10)
+
+    def forward(self, x):
+        x = self.conv1(x)
+        x = F.relu(x)
+        x = self.conv2(x)
+        x = F.relu(x)
+        x = F.max_pool2d(x, 2)
+        x = self.dropout1(x)
+        x = torch.flatten(x, 1)
+        x = self.fc1(x)
+        x = F.relu(x)
+        x = self.dropout2(x)
+        x = self.fc2(x)
+        output = F.log_softmax(x, dim=1)
```

Comment:
@alex-golts , is this log_softmax make sense here?

Reply:
@mosheraboh @SagiPolaczek I looked into it. I think it's okay that it's there. the logits before log_softmax are basically the same as after, up to a constant. so applying softmax on x, or on x after log_softmax, is the same.
I think applying the log_softmax can have numerical benefits when calculating CE or NLL loss after it, but I guess that in this simple example it probably doesn't matter much

Reply:
@alex-golts CE applies log_softmax and NLL loss.
In that case, you will apply log_softmax twice 
![image](https://user-images.githubusercontent.com/86309179/160396713-410dbfaf-162d-412a-99db-ff13641f6424.png)

Reply:
> @alex-golts CE applies log_softmax and NLL loss.
> In that case, you will apply log_softmax twice
> 

yes, but applying it twice doesn't change the result :-)

Reply:
@alex-golts 
Agree. But it's probably less numerically stable.
I'm approving, but if you agree it's less numerically stable, consider removing it.